Groza Dominic/L9_P3.2/main.c

```diff
@@ -15,10 +15,10 @@ int main() {
     int posi,len;
     char initial_String[30];
```

Comment:
Again, I have told you *so. many. times.* not to use statically allocated/sized arrays or strings. Yet, you are still using them. Please change this, thank you!